decidim-core/db/migrate/20161117083053_add_locales_settings.rb

```diff
@@ -0,0 +1,13 @@
+class AddLocalesSettings < ActiveRecord::Migration[5.0]
+  def up
+    add_column :decidim_organizations, :available_locales, :string, array: true, default: []
+    add_column :decidim_organizations, :default_locale, :string
```

Comment:
Since we haven't really deployed anything yet, maybe we could just modify the original migration?